client/galaxy/scripts/mvc/workflow/workflow-model.js

```diff
@@ -0,0 +1,42 @@
+define([
+    "mvc/base-mvc",
+], function( baseMVC ){
+/* global Backbone */
+// workflow model
+
+var logNamespace = 'workflow';
+//==============================================================================
+/** @class model for tool citations.
+ *  @name Citation
+ *  @augments Backbone.Model
+ */
+var WorkflowItem = Backbone.Model.extend( baseMVC.LoggableMixin ).extend({
+    _logNamespace : logNamespace,
+
+    urlRoot: '/api/workflows',
+
+    toJSON: function(){
+    // need to overwrite this as endpoint expects the 'workflow' key in payload
+    return {workflow : this.attributes};
+    },
+
+});
+
+var WorkflowCollection = Backbone.Collection.extend({
+    model: WorkflowItem,
+    url: '/api/workflows',
+
+    initialize: function () {
+        this.fetch();
```

Comment:
I am not sure this belongs to the model, would it be better served in view with the rest of the controller's logic?

Reply:
Makes sense, I'll have a look